src/main/resources/META-INF/plugin.xml

```diff
@@ -75,14 +73,12 @@ Detailed description at <a href="https://github.com/kornilova-l/FlameViewer">git
     </extensions>
 
     <extensions defaultExtensionNs="com.github.kornilovaL.flamegraphProfiler">
-        <profilerToFlamegraphConverter
+        <profilerToCompressedFlamegraphConverter
                 implementation="com.github.kornilova_l.flamegraph.plugin.server.converters.file.jmc.JMCConverter"/>
         <profilerToCompressedFlamegraphConverter
                 implementation="com.github.kornilova_l.flamegraph.plugin.server.converters.file.yourkit.csv.YourkitCsvToCFlamegraphConverter"/>
         <fileToCallTracesConverter
                 implementation="com.github.kornilova_l.flamegraph.plugin.server.converters.calltraces.yourkit.csv.YourkitCsvToCallTracesConverter"/>
-        <fileToCallTracesConverter
-                implementation="com.github.kornilova_l.flamegraph.plugin.server.converters.calltraces.flamegraph.FlamegraphToCallTracesConverter"/>
```

Comment:
This one is still used (we still do not convert flamegraph to cflamegraph when user uploads a file)

Reply:
Sorry, will re-add this file.